src/utils/efd.h

```diff
@@ -1,6 +1,7 @@
 /*
     Copyright (c) 2012-2013 Martin Sustrik  All rights reserved.
     Copyright 2015 Garrett D'Amore <garrett@damore.org>
+    Copyright (c) 2015-2016 Jack R. Dunaway.  All rights reserved.
```

Comment:
Just a heads up about All rights reserved.  This sentence means *nothing* in modern copyright law, and hasn't since 2000 (when the last hold out to Berne -- Nicaragua -- finally signed the Berne convention.)  That's why I never list it.

Similarly "(C)" has no meaning, and never has.  Its an attempt to replace &copy; with an ASCII-ifcation, but even &copy; means nothing in the presence of the word "copyright" or the abbreviation "copyr."  The symbolic form was intended to allow a short symbol *instead* of the word, rather than in addition to.  The ASCII version has never had any legal meaning whatsoever in a copyright notice, only the actual symbol or the word "copyright" does.

All of which is why my notices are always just "Copyright <year> <name>".  (I add my email too, but that's optional.)  The <year> is the most recent date of publication; adding the first year of publication is also optional but carries no particular weight of law behind it.